src/MoBi.Assets/AppConstants.cs

```diff
@@ -283,17 +283,17 @@ public static string DisplayValue(double displayValue, string displayUnit)
 
          public static string CommitingChangesToModulesMessage(ModuleConfiguration moduleConfiguration, bool hasMoleculeChanges, bool hasParameterChanges)
          {
-            string message = $"Changes will be applied in the module <i>{moduleConfiguration.Module.Name}</i>:{Environment.NewLine}";
+            string message = $"Changed parameter values will be committed to the parameter values of the module <i>{moduleConfiguration.Module.Name}</i>:{Environment.NewLine}";
```

Comment:
would extend the message with _...will be committed to the parameter values **building block** of the module..._